packages/cozy-pouch-link/src/replicateOnce.js

```diff
@@ -36,7 +36,8 @@ export const replicateOnce = async pouchManager => {
 
       const getReplicationURL = () => pouchManager.getReplicationURL(doctype)
 
-      const initialReplication = !pouchManager.isSynced(doctype)
+      const initialReplication =
+        pouchManager.getSyncStatus(doctype) !== 'synced'
```

Comment:
it might be worth to use constants for the different sync status

Reply:
This is not needed as the project's compiler will prevent to use the wrong values

Autompletion:
![image](https://github.com/user-attachments/assets/62410608-3978-4579-95e1-b13b33a21e1f)

IDE linter:
![image](https://github.com/user-attachments/assets/110c0156-e220-4d11-90bf-11b9703d5a8f)

CI error:
![image](https://github.com/user-attachments/assets/b347bfd0-dc98-4182-b4c8-b03b5dcadc36)